test/framework/src/main/java/org/elasticsearch/index/mapper/TextFieldFamilySyntheticSourceTestSetup.java

```diff
@@ -160,73 +157,7 @@ private String randomString() {
 
         @Override
         public List<MapperTestCase.SyntheticSourceInvalidExample> invalidExample() throws IOException {
-            Matcher<String> err = equalTo(
-                String.format(
-                    Locale.ROOT,
-                    "field [field] of type [%s] doesn't support synthetic source unless it is stored or"
-                        + " has a sub-field of type [keyword] with doc values or stored and without a normalizer",
-                    fieldType
-                )
-            );
-            return List.of(
-                new MapperTestCase.SyntheticSourceInvalidExample(err, b -> b.field("type", fieldType)),
-                new MapperTestCase.SyntheticSourceInvalidExample(err, b -> {
-                    b.field("type", fieldType);
-                    b.startObject("fields");
-                    {
-                        b.startObject("l");
-                        b.field("type", "long");
```

Comment:
Keep this?

Reply:
We now use fallback in any case that is not handled by the mapper explicitly. That means that text fields work even if they are not stored and don't have suitable keyword fields which is what this test tests.